app/presenters/queries/content_item_presenter.rb

```diff
@@ -125,6 +128,19 @@ def publication_state_sql
         SQL
       end
 
+      def state_history_sql
+        <<-SQL
+          (
+            SELECT json_agg((number, name))
+            FROM content_items c
```

Comment:
Also – if you know a better way to do this `SELECT`, I'm all ears. I try to avoid correlated subqueries if at all possible as (traditionally?) they've been terrible for performance.